keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/UnhealthyNotificationListener.kt

```diff
@@ -64,9 +82,9 @@ class UnhealthyNotificationListener(
 
     return Notification(
       subject = "${resource.spec.displayName} is unhealthy",
-      body = "<$resourceUrl|${resource.id}> is unhealthy and " +
-        "Spinnaker does not detect any changes to this resource. " +
-        "Manual intervention might be required. Please check the History view to see more details.",
+      body = "<$resourceUrl|${resource.id}> has been unhealthy for ${friendlyDuration(unhealthyDuration)} and " +
+        "Spinnaker cannot fix it. " +
+        "Manual intervention might be required. Please check the History view for more details.",
```

Comment:
@erikmunson I added the friendly duration back in. it's much better

Reply:
yeeeeeeess